dvc/command/metrics.py

```diff
@@ -37,17 +37,21 @@ class CmdMetricsShow(CmdBase):
     def run(self):
         typ = self.args.type
         xpath = self.args.xpath
-        try:
-            metrics = self.repo.metrics.show(
-                self.args.path,
-                typ=typ,
-                xpath=xpath,
-                all_branches=self.args.all_branches,
-                all_tags=self.args.all_tags,
-                recursive=self.args.recursive,
-            )
 
-            show_metrics(metrics, self.args.all_branches, self.args.all_tags)
+        try:
+            for target in self.args.targets:
```

Comment:
We should probably process all the targets we can, and only then return 1 if anything failed. E.g. see dvc/command/repro.py

Reply:
okok :+1: